### ISSUE TITLE
Export Chat Modal should not assume success until the chat is saved.

### ISSUE THREAD

**Issue by yaya-usman** [vector-im/element-web#22007]
#### What would you like to do?
I realized when i tried to export chat to my local machine, it assumes success and location it was saved in even when i ended up not saving the chat as demonstrated in the image below, The modal should only be in its current format if it doesn't prompt the user to save it in a specific location, else it's better it waits for it to be saved before it displays success, similarly if it's canceled it should probably just close the modal or show message like `failed to save`

![issueExport](https://user-images.githubusercontent.com/38439166/166159710-d87c7f00-d0c0-4302-bb0e-624ba3e0601c.PNG)

#### Why would you like to do it?
For smoother user experience and eradication of false positives.

### Have you considered any alternatives?

Mentioned in the description of problem above

**Comment by t3chguy** [comment]
> The modal should only be in its current format if it doesn't prompt the user to save it in a specific location, else it's better it waits for it to be saved before it displays success, similarly if it's canceled it should probably just close the modal or show message like failed to save

Browsers don't give apps access to information about downloads like this

**Comment by yaya-usman** [comment]
So there is no way we could know if the user saved the chat or not?

**Comment by yaya-usman** [comment]
Also, let's assume that's the case, the message on the modal seems to be a little off in the sense that, it shouldn't specify where the file is saved if it's gonna prompt the user to save it in their desired location, so i guess the original behavior was to just save the exported chat to downloads without prompting the user to save in a specific location.

**Comment by t3chguy** [comment]
> So there is no way we could know if the user saved the chat or not?

Correct, we ask the browser to initiate a download and that is the last we hear of it
